tests/golden_config_infra/test_config_reload_with_rendered_golden_config.py

```diff
@@ -12,12 +12,14 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('any'),
+    pytest.mark.topology('t0', 'm0', 'mx', 't1'),
     pytest.mark.disable_loganalyzer,
```

Comment:
@wen587  
Does the tested function on T2 is applicable for this test module?
If so, my team can help to update the test module to make it compatible with T2 test.
If not, could you use conditional mark to skip it on T2, so that we have a better control?

Reply:
We are planning to choose different deign. So this might not be kept in future.
It should be good so far. I added it back.